src/CityData.js

```diff
@@ -0,0 +1,49 @@
+export default [
```

Comment:
what's the purpose of this file?

Reply:
used this file to while creating UI to avoid frequent api requests. file now removed